.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: ruby-hack-assembler tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Install dependencies
+        run: gem install minitest
+      - name: Run tests
+        run: ruby -Ilib -e 'ARGV.each { |f| require f }' ./*_test.rb
```

Comment:
The `-Ilib` option means “put the `lib` directory on the load path (so that `require` knows where to look)”, which is pretty standard when running tests, but you don’t have a `lib` directory so you don’t need it. 

You actually don’t need to set up the load path at all because the `./*_test.rb` part provides full paths for all the files you’re going to `require` (and they themselves use `require_relative` so don’t make use of the load path), but the nearest equivalent would be:

```
ruby -I. -e 'ARGV.each { |f| require f }' *_test.rb
```

That is: add the current directory to the load path, and then `require 'assembler_test.rb'`, `require 'code_test.rb'` etc, all of which can be found on the load path.

Reply:
Ah good catch - fixed!

Reply:
Actually I just removed it completely `ruby -e 'ARGV.each { |f| require f }' ./*_test.rb`

Reply:
Yes, in my `-I` suggestion I had removed the `./` prefix from `./*_test.rb`, but if you keep the `./` prefix you don’t need `-I` at all.